Yank/repex.py

```diff
@@ -570,7 +570,7 @@ class ReplicaExchange(object):
     # Options to store.
     options_to_store = ['collision_rate', 'constraint_tolerance', 'timestep', 'nsteps_per_iteration', 'number_of_iterations', 'equilibration_timestep', 'number_of_equilibration_iterations', 'title', 'minimize', 'replica_mixing_scheme', 'online_analysis', 'show_mixing_statistics']
 
-    def __init__(self, store_filename, mpicomm=None, mm=None, **kwargs):
+    def __init__(self, store_filename, mpicomm=None, platform=None, mm=None, **kwargs):
```

Comment:
Same thing. If I put `platform` in `kwargs` in `ReplicaExchange`, it's going to be automatically included in the YAML syntax.